linux-anvil-cos7-cuda/Dockerfile

```diff
@@ -0,0 +1,111 @@
+ARG CUDA_VER
+
+FROM nvidia/cuda:${CUDA_VER}-devel-centos7
```

Comment:
I wonder if we can template this with a build `ARG` so that we can reuse the existing image a bit. Just trying to figure out how we can keep the maintenance burden low.

Reply:
yeah, I could try to template on centos version too, if you would like

Reply:
If it's not too difficult, otherwise we can skip it. I'm also happy to look as well :)

Reply:
yeah, working on it right now

Reply:
Thanks Anthony! Please let me know if you need a hand.

Reply:
Will do thanks John!